### ISSUE TITLE
Artist page banner title overlaid on band title in image

### ISSUE THREAD

**Issue by lahollen** [therabble/ccr-app#31]
Examples:
- [ ] http://ccr-public.herokuapp.com/artists/choir-and-marching-band
- [ ] http://ccr-public.herokuapp.com/artists/wilfred-n-and-the-grown-men

**Comment by mbrakken** [comment]
What result do we want here? It seems like a content problem, so EPL should request different images without text.

**Comment by prestonaustin** [comment]
@mbrakken I will request replacement art from Alex. We'll invite her and Peter shortly, and assign this task to her.

**Comment by acecarruthers** [comment]
I read through our license and realized it is within our rights to just crop the text out of these images (Section 2(g) says we can do whatever we want if we are acting reasonably!). I have cropped the offending text out and attached the images below. 

bannerccr.jpg is for Choir and Marching Band
wngm-band-banner-darker.jpg is for Wilfred N and the Grown Men

I've also uploaded these images to the Google Drive as a back up. Let me know if they work! 

![bannerccr](https://cloud.githubusercontent.com/assets/7355887/8582938/255683ae-2587-11e5-885f-3fbd61b9457b.jpg)
![wngm-band-banner-darker](https://cloud.githubusercontent.com/assets/7355887/8582943/2e61816a-2587-11e5-80c1-dc8678a5e2e6.jpg)
